### PR TITLE
accessibility findings in Toolbar

### PR DESCRIPTION
**Pull request by nurjinjafar**
as part of our accessibility improvements, we stumbled upon few things in toolbar buttons that could use some fixing to match the WCAG 2.0 cryteria:
the changes include:
- adding toggled labels for the audio and video buttons to show the correct tooltip and also beeing read correctly by the screen reader (that applies for prejoin toolbox as well)
screenshot(1):
Bildschirmfoto 2023-12-17 um 21 43 45

Bildschirmfoto 2023-12-17 um 21 43 04
 - on the prejoin screen when both camera and microphone are working properly, the message "your michrophone is working properly" always was shown even if the user is trying to activate the camera and was read by screen reader as well. which was a bit missleading and confusing to screen reader users, therefor we changed the text to "Everything is working properly". (we are open for other suggestions). see screenshot(1) above.

- for the drop down menu to join without audio next to join meeting button was not read correctly. Hence we tried to distinguish the label based on the state.

- and other fixes for the translation to match!

Contributors guide: https://github.com/jitsi/jitsi-meet/blob/master/CONTRIBUTING.md
-->

### CODE REVIEW

File: react/features/always-on-top/ToolbarButton.tsx [hunk 1]
```diff
@@ -54,7 +53,6 @@ const ToolbarButton = ({
     return (<div
         aria-disabled = { disabled }
         aria-label = { accessibilityLabel }
-        aria-pressed = { toggled }
```
Comment (Calinteodor) [inline react/features/always-on-top/ToolbarButton.tsx:57]: Why dropping this attribute?

File: react/features/prejoin/components/web/Prejoin.tsx [hunk 2]
```diff
@@ -432,8 +432,9 @@ const Prejoin = ({
                         visible = { showJoinByPhoneButtons }>
                         <ActionButton
                             OptionsIcon = { showJoinByPhoneButtons ? IconArrowUp : IconArrowDown }
-                            ariaDropDownLabel = { t('prejoin.joinWithoutAudio') }
-                            ariaLabel = { t('prejoin.joinMeeting') }
+                            ariaDropDownLabel = { showJoinByPhoneButtons
+                                ? t('prejoin.joinWithoutAudioMenuClose') : t('prejoin.joinWithoutAudioMenuOpen') }
+                            ariaLabel = { t('prejoin.joinWithoutAudio') }
```
Comment (Calinteodor) [inline react/features/prejoin/components/web/Prejoin.tsx:437]: I don't agree with changing the aria-label for this button. First, this button is for joining a meeting. Joining without audio is something that a user decides after interacting with the dropdown menu.

File: react/features/always-on-top/ToolbarButton.tsx (hunk 1 above)
Comment (nurjinjafar) [reply react/features/always-on-top/ToolbarButton.tsx]: it was because the previous state was always read as only "pressed" or "not pressed" as the button clicks on it. so i removed the aria-pressed from almost everywhere! anyhow i think where needed we can just add more statefull describtion. therefore, reverted!

**Comment by nurjinjafar** [comment]
@Calinteodor i applied the changes could you please take another look

File: react/features/always-on-top/ToolbarButton.tsx (hunk 1 above)
Comment (Calinteodor) [reply react/features/always-on-top/ToolbarButton.tsx]: I agree and understand. Not relevant because buttons are pressed to enable or disable something, except when the buttons themselves are disabled or not available. Let's remove this attribute.

**Comment by Calinteodor** [comment]
> @Calinteodor i applied the changes could you please take another look

Replied.

File: react/features/always-on-top/ToolbarButton.tsx (hunk 1 above)
Comment (Calinteodor) [reply react/features/always-on-top/ToolbarButton.tsx]: On second thought, let's not remove it. We have buttons that open panels like Chat or Participants. This is a case which requires knowing if a button was or wasn't pressed.

File: react/features/base/toolbox/components/ToolboxItem.web.tsx [hunk 3]
```diff
@@ -83,7 +83,7 @@ export default class ToolboxItem extends AbstractToolboxItem<IProps> {
             toggled
         } = this.props;
         const className = showLabel ? 'overflow-menu-item' : 'toolbox-button';
-        const buttonAttribute = isMenuButton ? 'aria-expanded' : 'aria-pressed';
+        const buttonAttribute = isMenuButton ? 'aria-expanded' : '';
```
Comment (Calinteodor) [inline react/features/base/toolbox/components/ToolboxItem.web.tsx:86]: Please revert this change as well.

**Review by Calinteodor — CHANGES_REQUESTED** [review]

File: react/features/base/toolbox/components/ToolboxItem.web.tsx (hunk 3 above)
Comment (mgcm) [reply react/features/base/toolbox/components/ToolboxItem.web.tsx]: picking this up so we can move forward - reverted per your request.

thanks!

**Review by Calinteodor — APPROVED** [review]

File: react/features/base/toolbox/components/ToolboxItem.web.tsx (hunk 3 above)
Comment (Calinteodor) [reply react/features/base/toolbox/components/ToolboxItem.web.tsx]: Thank you!

**Comment by Calinteodor** [comment]
Jenkins please test this please

**Comment by Calinteodor** [comment]
@nurjinjafar @mgcm There are some tests failing. You need to go https://github.com/jitsi/jitsi-meet-torture
create a branch with the same name as this one, run failing tests, apply fixes and create a PR.

**Comment by mgcm** [comment]
@Calinteodor I've opened a PR to fix the failing tests at https://github.com/jitsi/jitsi-meet-torture/pull/631

**Comment by Calinteodor** [comment]
Jenkins please test this please!
